modules/bloop-rifle/src/main/scala/scala/build/blooprifle/BloopRifleConfig.scala

```diff
@@ -119,6 +119,10 @@ object BloopRifleConfig {
       .orElse(fromProps)
       .getOrElse(hardCodedDefaultScalaVersion)
   }
+  lazy val extraTimeout: FiniteDuration = Option(System.getenv("BLOOP_EXTRA_TIMEOUT"))
```

Comment:
```suggestion
  lazy val extraTimeout: FiniteDuration = Option(System.getenv("SCALA_CLI_EXTRA_TIMEOUT"))
```

We can make this variable more name generic so that we can use this value in other places in the future.